packages/components/src/focal-point-picker/styles/focal-point-style.ts

```diff
@@ -3,45 +3,30 @@
  */
 import styled from '@emotion/styled';
 
-/**
- * WordPress dependencies
- */
-import { Path, SVG } from '@wordpress/primitives';
-
-/**
- * Internal dependencies
- */
-import { COLORS } from '../../utils';
-
 export const FocalPointWrapper = styled.div`
 	background-color: transparent;
 	cursor: grab;
-	height: 30px;
-	margin: -15px 0 0 -15px;
-	opacity: 0.8;
+	height: 48px;
+	margin: -24px 0 0 -24px;
 	position: absolute;
 	user-select: none;
-	width: 30px;
+	width: 48px;
```

Comment:
What do you think about using something like `clip-path: circle( 50% );` here as well? At the moment you can mouseover the invisible parts of the rectangle and still drag. It's quite nice when only the circle is draggable.

Reply:
We can try it!

Reply:
It causes issues with the shadow which is necessary for contrast in more varied backgrounds

Reply:
One way to make Dan’s suggestion work with the shadow is by putting the shadow and border-radius on the parent with `pointer-events: none` and the inner element gets the clip-path and `pointer-events: auto`.